### ISSUE TITLE
API inconsistency between parquet.votable and votable.parquet

### ISSUE THREAD

**Issue by bsipocz** [astropy/astropy#17257]
For votable.parquet we have `column_metadata` while for parquet.votable we have `metadata`.

Now, I have kept this inconsistency in #16375 as we have already run into the issue that the metadata we are writing is NOT column metadata in the parquet sense as we keep it all as metadata in one votable, so having the kwarg name as column_metadata could be rather confusing. But it in fact describes the columns themselves.
So, now we have this unfortunate API inconsistency with the other format. 

Should I go ahead and fix this while the new format and API is not yet released or we rather keep the inconsistency? It is also an option to rename the kwarg in votable.parquet to be  just `metadata`, but that will have to go through the deprecation cycle.

cc @afaisst @gpdf @tomdonaldson @delucchi-cmu

Milestoned to v7.0 as ideally this discussion should conclude parquet.votable is released.

**Comment by afaisst** [comment]
Can we make it so it takes the astropy metadata by default and adds the "column_metadata" if needed? Can we do "column_info"? (not sure I understand the problem correctly :D)

**Comment by pllim** [comment]
Branching has happened and RC will happen soon. Any chance this can be decided within a few days? If not, I will remove milestone and you can deal with it later as a breaking change. Thank you for your understanding.

**Comment by tomdonaldson** [comment]
My sense is that there are enough open issues with how that metadata is handled in the APIs (including some TODOs in #16375) that it might be preferable to address those en masse soon with possible breaking changes than to rush in this specific fix which may then be incompatible with whatever is decided for the TODOs.  If all the changes happen reasonably quickly after the IVOA meeting in 2 weeks, then I'm not concerned about the API in #16375 getting breaking changes.

**Comment by bsipocz** [comment]
Oh, this issue is just about the kwarg naming inconsistency. I think 'metadata' is better, but if we rather have it consistent with the existing writer we may want to change it before the release.

**Comment by tomdonaldson** [comment]
Yes, I was just imagining possible approaches to the other open items as possibly overlapping, e.g., splitting into multiple kwargs, conveying the information some other way, etc.  Maybe that's unlikely, but given that I agree that of the two `metadata` is better and that going that route with the older code is a breaking change, it doesn't seem like something we need to rush to solve.

**Comment by bsipocz** [comment]
One suggestion came out of a meeting today, calling it `vo_metadata`. We can then build validators that work with checking if the column describing metadata in the votdable part is consistent with what may end up being on the parquet columns, etc but at least on the kward level there won't be any confusions.

And then I would open a PR for the other reader and rename `column_metadata` to be `vo_metadata`, too. Would be great if such a PR could still be backported, but I suppose it can wait out one more release cycle if neccessary.
